yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -322,6 +326,68 @@ func checkAndParseEscapeAndQuoteChar() {
 
 }
 
+func packAndSendImportDataFilePayload(status string) {
+	if !callhome.SendDiagnostics {
+		return
+	}
```

Comment:
just thinking, instead of having `SendDiagnostics` flag check everywhere, can we let this functions run.
And at the end, where we send the payload, just check this flag and proceed based on that.

One problem with that approach i think is unnecessary processing if we don't need to send, but depends how costly that is....

What do you think?

Reply:
I think it's better to have this extra check as to prevent even packing the payload to avoid any unnecessary or unknown failures.